platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/StateFileManagerTests.java

```diff
@@ -480,9 +437,8 @@ void stateDeletionTest() throws IOException {
         }
     }
 
-    // FUTURE WORK: https://github.com/hiero-ledger/hiero-consensus-node/issues/19905
     private static void hashState(MerkleNodeState state) {
```

Comment:
I think we can get rid of this method in favor of inline comments OR use it more throughout tests and add JavaDoc for it